Assets/_Laz/Scripts/Managers/LazoWrappableManager.cs

```diff
@@ -42,11 +42,11 @@ private void SetupPlanetoids(IPlanetoidBehaviour[] planetoids)
                 planetoidBehaviour.Initialize();
                 var planetoid = planetoidBehaviour.PlanetoidModel;
                 _listOfPlanetoids.Add(planetoid);
-                planetoid.OnActivated += HandleOnWrappableActivated;
+                planetoid.OnPlanetoidActivated += HandleOnWrappablePlanetoidActivated;
```

Comment:
Should there be an unsubscribe somewhere in the script?

Reply:
it'll get unsubscribed when it get's nulled since it's not a Monobehaviour

Reply:
I set it up to unsubscribe.